src/ObiUtils.test.ts

```diff
@@ -37,9 +38,9 @@ describe('OBIutils', () => {
             [Suggestions=Talk to agent|Goodbye]
             \`\`\`
             `
-            const lgMap = new Map<string, LGItem>()
+            const lgMap: LGItem[] = []
             ObiUtils.addToLGMap(input, lgMap)
```

Comment:
I see now that you prserved type info here, but not the one above. What's different about the array above? Intentionally `any[]`